src/LazyRender.jsx

```diff
@@ -111,6 +114,19 @@ var LazyRender = React.createClass({
     });
   },
 
+  componentDidUpdate: function() {
+    //important to update the child height in the case that the children change(example: ajax call for data)
+    if(this.state.childHeight !== this.getChildHeight())
```

Comment:
Space before the opening bracket and curly braces around the if statement please